.travis.yml

```diff
@@ -9,5 +9,6 @@ install:
 
 script:
   - python -m pytest --cov=invoice tests/
+  - python -m mypy invoice/
```

Comment:
Never used mypy. Will read up on it's purpose.

Reply:
but you were using type annotations!
mypy is static type analysis. It makes sure that, e.g. if a function expects an int as parameter, you don't give a string or a float.
I think the only point of using type annotations in python is to run mypy (or any similar tool, if there is). Or to document interfaces.